internal/envoy/config.go

```diff
@@ -36,11 +36,11 @@ type ConfigWriter struct {
 	AdminPort int
 
 	// StatsAddress is the address that the /stats path will listen on.
-	// Defaults to 0.0.0.0.
+	// Defaults to 127.0.0.1 and is only enabled if StatsdEnabled is true.
```

Comment:
This needs to be a path that Prometheus can scrape. If we default to localhost then it won't be accessible. My idea was to create another listener, but on a different port that bound to `0.0.0.0`, this way we could allow Prometheus to scrape and it wouldn't be published over a public interface.